### DISCUSSION TITLE
How can I implement model?

### DISCUSSION THREAD

**Discussion by Byun-jinyoung** [pyg-team/pytorch_geometric#4305] [Q&A]
Dear pyg users,
I am a bit new to deep learning and have studied pyg  and its applications to my project.
Recently, I've read []https://arxiv.org/pdf/2106.05667.pdf which the suggested model's name is `GraphiT`.
I studied the paper code and wanted to implement it by using `pyg`. In GraphiT code, it used relative positional encoding tensor as one of the inputs generated by diffusion kernel.
But the problem what I considered is that the other inputs such as `masked_x, mask` with fixed length/size for each batch.
So I want to implement the model with pyg dataset/dataloader for different size tensors for each batch.

Could you please let me know how can I solve this problem?

Best regards
Thank you :)
